### ISSUE TITLE
Is this different to 3.5's physics interpolation?

### ISSUE THREAD

**Issue by elvisish** [lawnjelly/smoothing-addon#36]
Sorry this isn't an issue, but I've been wondering for a long time if this addon does something different to the built-in physics interpolation in 3.5? If not, should they be used together?

**Comment by Calinou** [comment]
> if this addon does something different to the built-in physics interpolation in 3.5

Yes, they work differently internally. However, the end result should be identical.

> If not, should they be used together?

No. You should use the add-on *or* the built-in interpolation, not both.

**Comment by elvisish** [comment]
`No. You should use the add-on or the built-in interpolation, not both.`

Thanks, that's very helpful! I'm interested in when it would be preferable to use one solution over the other, or if it makes no difference (if the results are the same)?

**Comment by Calinou** [comment]
> I'm interested in when it would be preferable to use one solution over the other, or if it makes no difference (if the results are the same)?

The built-in interpolation should be preferred for 3D. For 2D, you should keep using this add-on until Godot gets built-in 2D physics interpolation ([which is being worked on](https://github.com/godotengine/godot/pull/76252)).

**Comment by lawnjelly** [comment]
After 3.6 beta 4, core physics interpolation should ideally be used in preference to the addon as it should be faster and more accurate (it deals with pivots correctly).

However I will maintain the addon for 3.x, for compatibility with earlier Godot versions and existing projects. Also there are some differences between the addon and core interpolation.
